### PR TITLE
switch around shrinkwrap check

### PR DESCRIPTION
**Pull request by michaelneale**
cc @scherler @reviewbybees

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -10,9 +10,9 @@ node {
   docker.image('cloudbees/java-build-tools').inside {
     withEnv(['GIT_COMMITTER_EMAIL=me@hatescake.com','GIT_COMMITTER_NAME=Hates','GIT_AUTHOR_NAME=Cake','GIT_AUTHOR_EMAIL=hates@cake.com']) {
       try {
+        sh "node ./bin/checkshrinkwrap.js"
         sh "mvn clean install -B -DcleanNode -Dmaven.test.failure.ignore"
         sh "node ./bin/checkdeps.js"
```
Comment (scherler) [inline Jenkinsfile]: line 15 should be one line up as well

Comment (michaelneale) [inline Jenkinsfile]: @scherler no - that actually breaks as there is no node_modules yet - it has to run after the build

Comment (michaelneale) [inline Jenkinsfile]: ie you will see: 

```
[checkdeps_before_build] Running shell script
+ node ./bin/checkdeps.js
module.js:327
    throw err;
    ^

Error: Cannot find module '../blueocean-dashboard/node_modules/@jenkins-cd/design-language/package.json'
```

**Comment by michaelneale** [comment]
Best to run after it seems

**Pull request closed without merging**